### DISCUSSION TITLE
Help on setting up Primevue Admin Template into Nuxt 3

### DISCUSSION THREAD

**Discussion by donPuerto** [primefaces/community#35] [PrimeVue Templates] [answered]
Hi, I just bought Apollo admin template, however, its only for Vue 3, I tried to plug into Nuxt 3, unfortunately it didn't work as expected, As I check the documentation, its only for Vue 3 only. Any Help?

**Comment by ace-featherlight** [comment]
Hi, yes the template it's only for Vue 3 using Vite
But I think the integration is very simple, very easy, for example you can refer to [sakai-nuxt](https://github.com/primefaces/sakai-nuxt), you can follow the mechanism

But I think it would be better if the template provided a Nuxt 3 version or Next for React as well, it would be better :)

**Reply by donPuerto** [reply]
Sakai-Jonson kinda difficult to follow, sakai-nuxt kinda old, in summary got stuck in converting from vite to nuxt due to type error :)

Hoping Primevue team, could have Nuxt Apollo version available.

**Comment by cagataycivici** [comment]
Main issue is maintaining two versions, pure Vite and Nuxt. In react land, we only maintain NextJs versions. However Apollo is definitely worth the effort due to popularity, I'll discuss this next week with the team to see if we can roll out a Nuxt version next week, I think we can do this for Apollo.

 Having said that, converting a vite based template to Nuxt is not super complex until then. You have Sakai Vite and Sakai Nuxt as samples.

**Reply by ace-featherlight** [reply]
If main issue is will be maintaining two versions, pure Vite and Nuxt
I'll aggree if the next version iteration of primevue template is only using nuxt, for pure vite maybe buyer just refer to older version (since you say converting is not super complex for nuxt to vite or vice versa)

**Comment by donPuerto** [comment]
is the startup template available now? @cagataycivici :)

**Comment by stack-vue** [comment]
Does this include support for Nuxt 3??

**Comment by tugcekucukoglu** [comment]
Apollo Nuxt will be released once we complete the Nuxt Module for PrimeVue.

**Reply by stack-vue** [reply]
Thanks @tugcekucukoglu . Any tentative date plz ??

**Comment by donPuerto** [comment]
Is the template available now?

**Comment by tugcekucukoglu** [comment, marked as answer]
Thank you for waiting!

Apollo-Vue 5.0.0 is released. https://apollo.primevue.org/documentation

You could reach from store now.
